README.md

```diff
@@ -10,6 +10,8 @@ For a list of changes between versions, see the [CHANGELOG.md](https://github.co
 
 > A list of community-maintained packages is being collected. To include your package, see the [Contributing guidelines](./CONTRIBUTING.md#creating-high-quality-community-packages).
 
+- [snarkyjs-elgamal](https://github.com/Trivo25/snarkyjs-elgamal): A partially homomorphic encryption library for SnarkyJS based on Elgamal encryption. [npm](https://www.npmjs.com/package/snarkyjs-elgamal)
```

Comment:
oh wdyt using the changes I propose in https://github.com/o1-labs/snarkyjs/pull/857 
<img width="977" alt="image" src="https://user-images.githubusercontent.com/10067215/233639095-069d2e12-c56e-4219-8760-a667db998999.png">

Reply:
Hey @barriebyron, I personally prefer the style of listing packages that we use here:

![image](https://user-images.githubusercontent.com/20989968/233641570-1d6f3b8f-b970-4f39-aa08-e758df34ef06.png)

* It's more concise -- 1-2 lines per package, so it doesn't get long as quickly as if we used 4+ lines
* The "repository on Github" and "package on the public npm registry" feel self-explanatory, wouldn't repeat that for every listed package
* "contributed by" link doesn't add much because the GH repo link is already a sub-page of the repo owner: [https://github.com/Trivo25/snarkyjs-elgamal](https://github.com/Trivo25/snarkyjs-elgamal)

Reply:
The challenge for me was that the links in the suggested model didn't fulfill the self-describing goal. @mitschabaude  wdyt about:

<img width="565" alt="image" src="https://user-images.githubusercontent.com/10067215/233668801-6c2e4ab7-f1cc-4820-a3ee-7c4a47798002.png">